jekyll/_cci1/faq.md

```diff
@@ -26,13 +26,25 @@ Our **[status page](https://status.circleci.com/)** gives information on current
 
 Currently we support authentication with GitHub and Bitbucket. Please vote on relevant [Feature Request](https://discuss.circleci.com/c/feature-requests) if you want to show support for other version control systems.
 
+### What if I don’t want to include so many orgs when I am authorizing CircleCI?
+ 
+You can change the org access using OAuth App Restrictions on Github, see the [GitHub Help OAuth documentation](https://help.github.com/articles/about-oauth-app-access-restrictions/oauth) for instructions.
+ 
 ### I can’t give CircleCI the access to all my private repositories. What do I do?
-
-GitHub has only recently added the fine-grained permissions options, and
-we are still working on supporting them.
-
-In the meantime, the [suggested workaround]( {{ site.baseurl }}/1.0/github-security-ssh-keys/) is to create an additional user on GitHub with a limited set of permissions and use that account to
```

Comment:
This is still a good workaround. maybe change language to say "Another known workaround for private organizations is to create an additional user on GitHub with a limited set of permissions and use that account to perform builds on CircleCI."